evidence/create_build.go

```diff
@@ -0,0 +1,128 @@
+package evidence
+
+import (
+	"fmt"
+	"github.com/jfrog/jfrog-cli-artifactory/evidence/utils"
+	"github.com/jfrog/jfrog-cli-core/v2/utils/config"
+	coreConfig "github.com/jfrog/jfrog-cli-core/v2/utils/config"
+	"github.com/jfrog/jfrog-client-go/artifactory"
+	"github.com/jfrog/jfrog-client-go/artifactory/services"
+	"github.com/jfrog/jfrog-client-go/utils/errorutils"
+	"github.com/jfrog/jfrog-client-go/utils/log"
+	"strings"
+)
+
+type createEvidenceBuild struct {
+	createEvidenceBase
+	project string
+	build   string
+}
+
+func NewCreateEvidenceBuild(serverDetails *coreConfig.ServerDetails, predicateFilePath string, predicateType string, key string, keyId string,
+	project string, build string) Command {
+	return &createEvidenceBuild{
+		createEvidenceBase: createEvidenceBase{
+			serverDetails:     serverDetails,
+			predicateFilePath: predicateFilePath,
+			predicateType:     predicateType,
+			key:               key,
+			keyId:             keyId,
+		},
+		project: project,
+		build:   build,
+	}
+}
+
+func (c *createEvidenceBuild) CommandName() string {
+	return "create-build-evidence"
+}
+
+func (c *createEvidenceBuild) ServerDetails() (*config.ServerDetails, error) {
+	return c.serverDetails, nil
+}
+
+func (c *createEvidenceBuild) Run() error {
+	artifactoryClient, err := c.createArtifactoryClient()
+	if err != nil {
+		log.Error("failed to create Artifactory client", err)
+		return err
+	}
+	subject, err := c.buildBuildInfoSubjectPath(artifactoryClient)
+	if err != nil {
+		return err
+	}
+	envelope, err := c.createEnvelope(subject)
+	if err != nil {
+		return err
+	}
+	err = c.uploadEvidence(envelope, subject)
+	if err != nil {
+		return err
+	}
+
+	return nil
+}
+
+func (c *createEvidenceBuild) buildBuildInfoSubjectPath(artifactoryClient artifactory.ArtifactoryServicesManager) (string, error) {
+	build := strings.Split(c.build, ":")
```

Comment:
Please, raise the question about the input parameters one more time.
This split means that neither Build Name nor Build Number may contain `:` (this is not the case).
Hence, it means that we do not support Builds with name/number having `:`.
Needs to be confirmed explicitly.

Reply:
I talked with Noam, there is a chance to change all other operations to be splitter to --name --version.
will take it in a separate Jira ticket.